app/admin/users.rb

```diff
@@ -1,11 +1,12 @@
 ActiveAdmin.register User do
-  permit_params :email, :password, :password_confirmation
+  permit_params :email, :admin, :partner_id, :password, :password_confirmation
```

Comment:
Thanks Andrew! You are a real treasure. Not a sr treasure, just a normal treasure.